### DISCUSSION TITLE
non Relative URLs

### DISCUSSION THREAD

**Discussion by userdocs** [mondeja/mkdocs-include-markdown-plugin#118] [Q&A] [answered]
Hi and thank for the plugin, just a quick question. Is there a root context to urls.

The docs have 

```
{%
   include-markdown "docs/includes/header.md"
   heading-offset=1
%}
```

But any attempt i have made to use urls relative to docs is not working.  So like this

```
include-markdown "docs/../includes/header.md"
include-markdown "docs/header.md"
```

The this example is not relative to `docs/`

https://github.com/mondeja/mkdocs-include-markdown-plugin/blob/master/examples/basic/docs/index.md

Is it possible to make urls relative to the top level or are they all relative to the file that includes them?

**Comment by mondeja** [comment]
> Is it possible to make urls relative to the top level or are they all relative to the file that includes them?

Nope. Because you can't know what are you specifying with `/docs/includes/header.md`, is `/docs` an absolute path to a folder relative to the storage prefix or is the prefix in where the docs are being served? Even absolute links don't work within Mkdocs, see https://github.com/mkdocs/mkdocs/pull/1623

**Comment by mondeja** [comment, marked as answer]
This is outdated because now we can include relative URLs. Thanks for the feedback!
